rollup.config.js

```diff
@@ -7,22 +7,21 @@ import { terser } from 'rollup-plugin-terser';
 const BASE_PATH = process.cwd();
 
 module.exports = [
-  "oasOp2xxResponse",
-  "oasOpFormDataConsumeCheck",
-  "oasOpIdUnique",
-  "oasOpParams",
-  "oasOpSecurityDefined",
-  "oasPathParam"
+  'oasOp2xxResponse',
+  'oasOpFormDataConsumeCheck',
+  'oasOpIdUnique',
+  'oasOpParams',
+  'oasOpSecurityDefined',
+  'oasPathParam',
+  'refSiblings'
 ].map(fn => ({
   input: path.resolve(BASE_PATH, 'dist/rulesets/oas/functions', `${fn}.js`),
   plugins: [
     typescript({
       tsconfig: path.join(BASE_PATH, './tsconfig.rollup.json'),
       include: ['dist/**/*.{ts,tsx}'],
     }),
-    resolve({
-      only: ['json-schema-merge-allof', /lodash\/?.*/],
```

Comment:
I'm not sure what `json-schema-merge-allof` but is it a dependency we can remove now?

Reply:
@philsturgeon actually it's a leftover :laughing: 
I copied that rollup config from json-schema-viewer repo and forgot to remove it.
We don't have that package included in our package.json, so we should be okay.